lib/mockbuilder.py

```diff
@@ -114,13 +132,25 @@ def _build_rpm(self, package):
             raise
 
     def prepare_sources(self, package):
+        """
+        Prepare package source files
```

Comment:
Perhaps we can briefly explain what actions are done in order to prepare the sources.  Example:

```python
def prepare_sources(self, package):
    """
    Create directory structure, create a tarball with the source code,
    and copy files to chroot.
    """
```